### DISCUSSION TITLE
Deploy with 500: INTERNAL_SERVER_ERROR

### DISCUSSION THREAD

**Discussion by LucaFrisoni** [vercel/community#3511] [Help]
### Summary

This Serverless Function has crashed.

Your connection is working correctly.

Vercel is working correctly.

500: INTERNAL_SERVER_ERROR
Code: FUNCTION_INVOCATION_FAILED
ID: gru1::f985h-1691251532303-642ea82c331d

### Example

https://django-project-zeta-blond.vercel.app/

### Steps to Reproduce

Please help me

**Comment by fabricio-fn** [comment]
Check environment variables if your function depends on environment variables

**Reply by LucaFrisoni** [reply]
No, it no depends on environment variables

**Comment by amyegan** [comment]
You should be able to get more info from the [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#). You can also wrap sections of code in a [try and except block](https://realpython.com/python-exceptions/#the-try-and-except-block-handling-exceptions) to log additional error details.

**Reply by LucaFrisoni** [reply]
Hey, the problem is that the runtime Logs give me this information Unknown application error occurred Runtime.Unknown.
And locally evertyhon works just fine, but the deploy has that error, dont know why
